test/integration/consumer_group_test.exs

```diff
@@ -211,4 +211,16 @@ defmodule KafkaEx.ConsumerGroup.Test do
 
     assert 0 >= offset_fetch_response_offset
   end
+
+  test "can join a consumer group" do
+    random_group = generate_random_string
+    random_topic = generate_random_string
+    KafkaEx.create_worker(:join_group, [uris: uris, consumer_group: random_group])
+    KafkaEx.produce(%Proto.Produce.Request{topic: random_topic, partition: 0, required_acks: 1, messages: [%Proto.Produce.Message{value: "hey"}]})
```

Comment:
I thought that might be the case, but you're creating a topic with a random name but then using `["foo", "bar"]` as the list of topics below.  Presumably the tests pass, so I'm wondering if the topics really need to be created?  Does the spec say this anywhere?

(If this does end up being the case, I'd say it's worth a comment in the test code because following all of the states that the broker/client needs to be in is pretty complex)